custom-prom.md

```diff
@@ -67,6 +67,29 @@ Kubecost requires the following minimum versions:
 
 This config needs to be added to `extraScrapeConfigs` in the Prometheus configuration. Example [extraScrapeConfigs.yaml](./images/extraScrapeConfigs.yaml)
 
+<a name="recording-rules"></a>
```

Comment:
```suggestion
## Recording rules
```

Reply:
Proper MD syntax

Reply:
but it isn't a new section. it is part of #2

Reply:
Just correct it to ###. The link I posted above will still work to direct users to this anchor

Reply:
like this?